packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputElement.tsx

```diff
@@ -1,15 +1,16 @@
-import { forwardRef, ReactNode } from 'react';
+import React, { forwardRef, ReactNode } from 'react';
```

Comment:
You are the one who removed all React imports from the codebase, right @AdamSchinzel ? 😃  https://github.com/trezor/trezor-suite/pull/9331

Reply:
I guess it might have been a snippet :DD I edited my component snippet today to avoid that.